### DISCUSSION TITLE
Question-is it possible to trigger a script upon guest link expiration

### DISCUSSION THREAD

**Discussion by dkhan717** [Ylianst/MeshCentral#5835] [Q&A]
A little background - I have already installed the plugin "Scriptask" and I usually run a Hyper-V reset PowerShell script once my client guest link has expired. However, I do this manually, and I have to do it on a weekly basis, which is becoming tedious as we have different clients. Is it possible to trigger a specific PowerShell script automatically when a guest link expires? For example, suppose I share "device1" for 24 hours, and once the link expires, it triggers the script from the "Scripttask" plugin on that device.

**Comment by si458** [comment]
Plugins are not officially supported so we cannot offer full support/help
However as far as im aware the is no way of knowing when a link is expired but I would have to look into it

**Reply by dkhan717** [reply]
Thank you any info would be great appreciate your time.
